main.go

```diff
@@ -1,21 +1,36 @@
 package main
 
+import "flag"
```

Comment:
move this into the import() block below. Atom should do this for you with the correct linters enabled.